### ISSUE TITLE
Picker style

### ISSUE THREAD

**Issue by qaliwebguru** [UXDivers/Grial-UI-Kit-Support#272]
I am not sure if there is any picker style available with bottom border?
![dropdown-style](https://user-images.githubusercontent.com/31895946/31375476-9aa2575e-ad99-11e7-872a-f56ba67b3a38.png)

**Comment by LeoHere** [comment]
Currently we only support styling the picker in iOS. But we are workin on it. I will let you know as soon we have a version you can try.

**Comment by LeoHere** [comment]
@qaliwebguru we have just published and update of the Nuget packages (2.5.6-RC) which includes updates the `Picker`/`DatePicker`/`TimePicker` renderers to support the `PickerProperties` in Android. 

These properties are:

~~~
artina:PickerProperties.BorderStyle="BottomLine" 
artina:PickerProperties.BorderWidth="2" 
artina:PickerProperties.BorderColor="Red" 
artina:PickerProperties.PlaceholderColor="Red" 
artina:PickerProperties.BorderCornerRadius="10"
~~~

Previously these properties were only supported in iOS. 

You will need to define an style for the `Picker` 

~~~
<Style x:Key="PickerStyle" TargetType="Picker">
    <Setter Property="artina:PickerProperties.BorderStyle" Value="BottomLine" />
    <Setter Property="artina:PickerProperties.BorderWidth" Value="1" />
    <Setter Property="artina:PickerProperties.BorderColor" Value="Red" />
</Style>
~~~

and associate to the pickers in your page.

Notice the nugets are Release Candidate packages, so you will need to check the Show pre-release packages in order to see them.

**Comment by qaliwebguru** [comment]
Thanks a lot.
